README.md

```diff
@@ -74,6 +74,86 @@ jobs:
 
 [wde]: https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow
 
+## Trusted publishing
+
+To configure a npm token-less workflow, you can enable [Trusted publishing](https://docs.npmjs.com/trusted-publishers) on your npm package. Since this requires specifying a single workflow file, you'll need to create a shared release workflow that uses alternate triggers and conditions.
```

Comment:
```suggestion
To configure a npm token-less workflow, you can enable [Trusted publishing](https://docs.npmjs.com/trusted-publishers) on your npm package.
Since this requires specifying a single workflow file, you'll need to create a shared release workflow that uses alternate triggers and conditions.
```